plot/plotcore/barplot.go

```diff
@@ -35,16 +32,16 @@ func (pl *PlotEditor) genPlotBar() {
 	}
 	xp := pl.Columns[xi]
 
-	var lsplit *table.Splits
+	// var lsplit *table.Splits
 	nleg := 1
 	if pl.Options.Legend != "" {
-		_, err = pl.table.Table.ColumnIndex(pl.Options.Legend)
-		if err != nil {
-			log.Println("plot.Legend: " + err.Error())
+		lcol := pl.table.ColumnIndex(pl.Options.Legend)
+		if lcol < 0 {
+			log.Println("plot.Legend not found: " + pl.Options.Legend)
 		} else {
-			xview.SortColumnNames([]string{pl.Options.Legend, xp.Column}, table.Ascending) // make it fit!
-			lsplit = split.GroupBy(xview, pl.Options.Legend)
-			nleg = max(lsplit.Len(), 1)
+			// xview.SortColumnNames([]string{pl.Options.Legend, xp.Column}, tensor.Ascending) // make it fit!
+			// lsplit = split.GroupBy(xview, pl.Options.Legend)
+			// nleg = max(lsplit.Len(), 1)
```

Comment:
Why are there so many things commented out in this function? What is your plan? Can you just get rid of them?

Reply:
i will come back to that code later; uses new stats.Group code into datafs and it requires just a bit more time than I have now.. :)